app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2> Contestants Index </h2>
+
+  <% @contestants.each do |contestant| %>
+    <%= contestant.name %>
+    <p>Projects: <% contestant.projects.each do |project| %></p>
```

Comment:
You can do a nested iteration like this but you could also create a method in your `Contestant` model that plucks the names of all the projects and then call that method on line 5 vs iterating. 
```ruby
def projects_list
  projects.pluck(:name).to_sentence
end
```